File: src/kyu_8/collatz_conjecture_3n_plus_1/python/collatz_conjecture_3n_plus_1_test.py
import codewars_test as test
from solution import hotpo

@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(hotpo(1), 0);
        test.assert_equals(hotpo(5), 5);
        test.assert_equals(hotpo(6), 8);
        test.assert_equals(hotpo(23), 15);

File: src/kyu_8/collatz_conjecture_3n_plus_1/cpp/collatz_conjecture_3n_plus_1_test.cpp
Describe(Test){
    It(Examples){
	      Assert::That(hotpo(1), Equals(0));
        Assert::That(hotpo(5), Equals(5));
        Assert::That(hotpo(6), Equals(8));
        Assert::That(hotpo(23), Equals(15));
	  }
};